lib/down/httpx.rb

```diff
@@ -19,7 +19,7 @@ def initialize(**options, &block)
       headers = options.delete(:headers) || {}
       @client = HTTPX
           .plugin(:follow_redirects, max_redirects: 2)
-          .plugin(:basic_authentication)
+          .plugin(:basic_auth)
```

Comment:
I would like to still support HTTP 0.x versions if possible, for backwards compatibility. Would it be possible to add a conditional with 0.x and 1.x branches?

Reply:
That's reasonable,  I'll do the change 👍

Reply:
done